### DISCUSSION TITLE
Disabling sorting when replicating

### DISCUSSION THREAD

**Discussion by jerauf** [freekmurze/eloquent-sortable#156] [Ideas]
In Laravel 10 with eloquent-sortable 4.0, when using replicate, I want to leave the order column the same. However, it always updates since I'm creating.

Any way to disable sort_when_creating temporarily?

**Comment by freekmurze** [comment]
This currently isn't supported by the package. You'll need to update the order column manually.

**Comment by jerauf** [comment]
Thank you, @freekmurze.
